MessageKit.podspec

```diff
@@ -11,17 +11,11 @@ Pod::Spec.new do |s|
    s.source = { :git => 'https://github.com/MessageKit/MessageKit.git', :tag => s.version }
    s.source_files = 'Sources/**/*.swift'
 
-   s.pod_target_xcconfig = {
```

Comment:
Can you speak to these two removals and why they are not needed anymore?

Reply:
`s.requires_arc` is true by default - not necessary then.
`s.pod_target_xcconfig` SWIFT_VERSION` is legacy thing that was replaced by `s.swift_version` that we already have. More updates coming! :)